db/schema.yml

```diff
@@ -4541,6 +4541,17 @@ middleware_domains:
 - ems_id
 - created_at
 - updated_at
+middleware_messagings:
```

Comment:
Correct, `schema.yml` needs to be committed. `schema.rb` should not be committed.